debian/copyright

```diff
@@ -0,0 +1,34 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ndn-cxx
+Source: http://named-data.net
+
+Files: *
+Copyright: 2010-2016 Alexander Afanasyev   <http://lasr.cs.ucla.edu/afanasyev/index.html>
+           2010-2016 Yingdi Yu             <http://irl.cs.ucla.edu/~yingdi/>
+           2010-2016 Junxiao Shi           <http://www.cs.arizona.edu/people/shijunxiao/>
+           2010-2016 Jeff Thompson         <jefft0@remap.ucla.edu>
+           2010-2016 Zhenkai Zhu           <http://irl.cs.ucla.edu/~zhenkai/>
+License: LGPL-3.0+
+
+Files: debian/*
+Copyright: 2016 Endless Computers INC
+Copyright: 2016 Niv Sardi <xaiki@endlessm.com>
```

Comment:
Hmm... might be good to get @cosimoc's opinion on this but I _think_ the work is "authored by Endless" since it's work-for-hire and as such falls under its copyright
.
https://www.copyright.gov/circs/circ09.pdf
http://saperlaw.com/2007/09/27/five-things-every-software-developer-should-know/ under "Who is the author and who is the owner?"